packages/core/ai_content_pipeline/ai_content_pipeline/cli/output.py

```
@@ -0,0 +1,133 @@
+                "command": command,
+                "items": rows,
+                "count": len(rows),
+            }
+            print(json.dumps(envelope, default=str, indent=2))
+        elif not self.quiet:
+            if headers:
+                print("  ".join(f"{h:<20}" for h in headers))
+                print("-" * (22 * len(headers)))
+            for row in rows:
+                if isinstance(row, dict):
+                    print("  ".join(f"{str(v):<20}" for v in row.values()))
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)